### PR TITLE
Chore/update metadata remover

### PR DESCRIPTION
**Pull request by jduncan-xoi**
- switch to Typescript
- switch to Vite and Vitest
- use a logger vs console.logs

### CODE REVIEW

**Comment by Aleclindner** [comment]
First of all how dare you

**Comment by jduncan-xoi** [comment]
> First of all how dare you

The legend is alive!

**Comment by danielxoi** [comment]
@Aleclindner might as well review this since you're already here

File: package.json [hunk 1]
```diff
@@ -1,53 +1,46 @@
...
-    "babel-plugin-transform-remove-console": "^6.9.4",
-    "flow-bin": "^0.109.0",
-    "flow-copy-source": "^2.0.9",
     "ffprobe": "^1.1.0",
-    "ffprobe-static": "^3.0.0",
+    "ffprobe-static": "^3.1.0",
     "gm": "^1.23.1",
     "hasha": "^5.2.0",
     "is-corrupted-jpeg": "^1.0.2",
-    "jest": "^26.6.3",
```
Comment (calebgregory) [inline package.json:39]: this diff hits like a warm shower after a long day of working outside in the sun.

File: src/imageGpsExifRemover.ts [hunk 2]
```diff
@@ -17,27 +17,29 @@ const END_OF_PNG_TAG = 0x49454e44
...
 
-const killGPS
-= async (startOffset: number, littleEndian: boolean, read: ReadFunction, write: WriteFunction) => {
+const killGPS = async (startOffset: number, littleEndian: boolean, read: ReadFunction, write: WriteFunction) => {
   const entryNumberDataView = await readNextChunkIntoDataView(2, startOffset, read)
   const entries = entryNumberDataView.getUint16(0, littleEndian)
   const entryOffset = startOffset + 2
   // kill 'em all
-  console.log('writing 0s on gps data...', entries, entryOffset)
+  Logger.debug('writing 0s on gps data...', entries, entryOffset)
```
Comment (Aleclindner) [inline src/imageGpsExifRemover.ts:29]: might be a good idea to just take the logs out at this point, I was mostly just using them to figure out how this stuff worked when I was writing it

**Review by Aleclindner — APPROVED** [review]

**Comment by Aleclindner** [comment]
Approved and left one optional comment

**Comment by Aleclindner** [comment]
also rename this to privilege/update metadata remover

**Review by danielxoi — APPROVED** [review]

**Pull request merged**